bigtable/autoscaler/strategies_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for autoscaler.py"""
+
+from strategies import DOWNSCALE_STRATEGIES
+from strategies import UPSCALE_STRATEGIES
+
+
+def test_downscale():
+    downscale_strategy = DOWNSCALE_STRATEGIES['incremental']
+    assert downscale_strategy(5) == 3
```

Comment:
How do you run these tests without using the `unittest` module? Are they imported somewhere into a main file which runs all the tests?